src/main/java/edu/cornell/kfs/module/purap/service/impl/CuElectronicInvoiceHelperServiceImpl.java

```diff
@@ -1666,4 +1686,92 @@ protected void addRejectReasonsToNote(String rejectReasons, ElectronicInvoiceRej
         }
     }
 
+    /**
+     * Temporary override to work around an issue where input files with unsupported "xmlns:*" attributes
+     * could not be parsed successfully, but otherwise behaves the same as in the superclass.
```

Comment:
It looks like the function call to removeUnsupportedXmlnsAttributes is the only thing different from the parent's implementation.  I would suggest noting that in the comment.

Reply:
Good point, I will update the comment accordingly.